linux/just_files/new_just

```diff
@@ -474,97 +479,117 @@ function write_dockerfile()
-            ## - The GDAL Docker recipe will build the GDAL C Library for you
-            ## - GDAL needs numpy installed before it is built, otherwise, numpy
-            ##   integration will not be compiled
+            #T# Simple packages can be added as dependencies to your project by:
+            #T# - Running the container and installing them directly with pipenv; e.g.,
+            #T#     just pipenv install scipy
+            #T#     # installs scipy (Note: all other packages will be updated. Avoiding
+            #T#     # this is difficult; the --keep-outdated flag is currently buggy)
+            #T#     just sync # (optionally updates the docker image)
+            #T# - Or editing your Pipfile manually (https://pipenv.pypa.io/en/latest/advanced/)
+            #T#   and then run
+            #T#     just sync
+
+            #T# GDAL, for example, is a more complicated example
+            #T# - The GDAL Docker recipe will build the GDAL C Library for you
+            #T# - The GDAL pypi packages needs numpy installed before it is built,
+            #T#   otherwise, numpy integration will not be compiled and
+            #T#   "import osgeo.gdal_array" will fail
             ##
-            ## To test this out:
-            ## 1) Uncomment all the "Uncomment for GDAL" sections
-            ## 2) just sync
-            ## 3) Remove the existing pipenv install section below
-            ## 4) Edit your Pipfile and add the following lines (or similar) to the
-            ##    [packages] section
-            ##        gdal = "==2.1.0"
-            ##        numpy = "*"
-            ##    This will add the latest version of numpy and the version of
-            ##    gdal compatible with debian:stretch to your pipenv environment.
-            ##    Note: the version of the pypi package should match (as closely as
-            ##    possible) to the version of the GDAL-binary dependency (gdal-bin)
-            ## 5) just pipenv install
-            ## 6) just sync (optionally update docker image)
-            ##
-            ## Steps 4 and 5 can also be done by:
-            ## - just pipenv install numpy gdal==2.1.0
-            ## - These can be combined because numpy is already installed at
-            ##   just sync time, so numpy was already installed before gdal
-            ## - All other pipenv install arguments can be passed in, such as
-            ##   --pre, etc...
+            #T# To test this out:
+            #T# 1) Uncomment all the "Uncomment for GDAL" sections
+            #T# 2) just sync # Update the image
+            #T# 3) just pipenv install numpy gdal==3.2.1 # Or edit the Pipfile manually
+            #T#    This will add the latest version of numpy and the version of the
+            #T#    pypi package gdal compatible with the GDAL C Library from the recipe.
+            #T#    Note: the version of the pypi GDAL and C GDAL should match as closely as possible
```

Comment:
```suggestion
            #T#    This will add the latest version of numpy (although a specific version can
            #T#    be specified) and a version of the pypi package GDAL compatible with the
            #T#    GDAL C Library from the recipe.
            #T#    Note: the version of the pypi GDAL and C GDAL should match as closely as
            #T#    possible
```